### PR TITLE
changed classspath to include ctrlib-sources

### PR DESCRIPTION
**Pull request by averycowan**

### CODE REVIEW

**Comment by tylkahn** [comment]
👍

**Review by tylkahn — APPROVED** [review]
Meh

File: .classpath [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
+    <classpathentry kind="var" path="wpilib" sourcepath="wpilib.sources"/>
+    <classpathentry kind="var" path="networktables" sourcepath="networktables.sources"/>
+    <classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+    <classpathentry kind="var" path="USERLIBS_DIR/navx_frc.jar" sourcepath="USERLIBS_DIR/navx_frc-sources.jar"/>
+    <classpathentry kind="var" path="USERLIBS_DIR/CTRLib.jar" sourcepath="/USERLIBS_DIR/CTRLib-sources.jar">
+        <attributes>
+            <attribute name="javadoc_location" value="http://www.ctr-electronics.com/downloads/api/java/html/index.html"/>
+        </attributes>
+    </classpathentry>
+    <classpathentry kind="var" path="opencv" sourcepath="opencv.sources"/>
```
Comment (c0d3rman) [inline .classpath]: Remove this

File: .classpath [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
+    <classpathentry kind="var" path="cscore" sourcepath="cscore.sources"/>
```
Comment (c0d3rman) [inline .classpath]: Remove this

File: .classpath [hunk 3]
```diff
@@ -1,10 +1,18 @@
...
+    <classpathentry kind="var" path="USERLIBS_DIR/CTRLib-javadoc.jar"/>
+    <classpathentry kind="var" path="USERLIBS_DIR/navx_frc-sources.jar"/>
```
Comment (c0d3rman) [inline .classpath]: Remove this

File: .classpath [hunk 4]
```diff
@@ -1,10 +1,18 @@
...
+    <classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+    <classpathentry kind="var" path="USERLIBS_DIR/navx_frc.jar" sourcepath="USERLIBS_DIR/navx_frc-sources.jar"/>
+    <classpathentry kind="var" path="USERLIBS_DIR/CTRLib.jar" sourcepath="/USERLIBS_DIR/CTRLib-sources.jar">
+        <attributes>
+            <attribute name="javadoc_location" value="http://www.ctr-electronics.com/downloads/api/java/html/index.html"/>
+        </attributes>
+    </classpathentry>
+    <classpathentry kind="var" path="opencv" sourcepath="opencv.sources"/>
+    <classpathentry kind="var" path="cscore" sourcepath="cscore.sources"/>
+    <classpathentry kind="var" path="USERLIBS_DIR/CTRLib-javadoc.jar"/>
```
Comment (c0d3rman) [inline .classpath]: Does this work?

**Review by c0d3rman — APPROVED** [review]

**Review by c0d3rman — CHANGES_REQUESTED** [review]
a

**Review by c0d3rman — APPROVED** [review]

**Pull request merged**